test/screenshot/mdc-fab/classes/baseline.html

```diff
@@ -1,46 +1,47 @@
 <!DOCTYPE html>
 <!--
-  ~ Copyright 2018 Google Inc. All Rights Reserved.
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~      http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
+  Copyright 2018 Google Inc. All rights reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why is this being changed when the Apache license itself lists `http:` (and so do the rest of our files)?

Reply:
All the other test pages - and all of our old demo pages - use `https://`.

I just copy-pasted this new version from an mdc-button test page, so at least all of the test pages will be consistent.